src/ess/offspec/load.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2025 Scipp contributors (https://github.com/scipp)
+import scipp as sc
+
+from ..reflectometry.types import Filename, RawDetectorData, ReferenceRun, RunType
+from .types import CoordTransformationGraph, MonitorData
+
+
+def load_offspec_events(
+    filename: Filename[RunType],
+) -> RawDetectorData[RunType]:
+    full = sc.io.load_hdf5(filename)
+    da = full['data']
+    da.coords['theta'] = full.pop('Theta')[-1].data
+    da = da.bins.concat('tof')
+    return da
+
+
+def load_offspec_monitor(
+    filename: Filename[RunType],
+    graph: CoordTransformationGraph[ReferenceRun],
+) -> MonitorData[RunType]:
+    full = sc.io.load_hdf5(filename)
+    mon = full["monitors"]["monitor2"]["data"].transform_coords(
```

Comment:
Please make the monitor name a parameter. See, e.g., cell 5 in https://scipp.github.io/esssans/user-guide/isis/sans2d.html